SeleniumWebDriverBasics/SeleniumWebDriverBasics/WebDriver/BrowserList.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SeleniumWebDriverBasics.WebDriver
+{
+    public abstract class BrowserList
```

Comment:
Is it necessary for BrowserList to be abstract? It doesn't have any sub-classes.